tracekit.js

```diff
@@ -235,8 +235,18 @@ TraceKit.report = (function reportModuleWrapper() {
         throw ex; // re-throw to propagate to the top level (and cause window.onerror)
     }
 
+    /**
+     * Returns whether there are any report handlers.
+     * @return {Boolean} whether there's any handlers
+     */
+    function hasHandlers() {
+        return handlers.length > 0;
+    }
+
+
     report.subscribe = subscribe;
     report.unsubscribe = unsubscribe;
+    report.hasHandlers = hasHandlers;
```

Comment:
Can we remove this logic, as it adds to the overall size of the library and seems to only be useful for the check below.